content/api/included-libraries.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Included Libraries
+---
+
+Python already comes with plenty of [built-in libraries](https://docs.python.org/3.4/py-modindex.html) to use, However there are some extra ones installed which may be of us:
```

Comment:
Once #45 is merged could `https://docs.python.org/3.4/` be replaced with the python_version variable?

Reply:
'may be of us' should be 'may be of use' ? also 'However' should not be capitalised after a comma